vms/platformvm/block/executor/standard_block_test.go

```diff
@@ -129,7 +129,6 @@ func TestBanffStandardBlockTimeVerification(t *testing.T) {
 	env.blkManager.(*manager).lastAccepted = parentID
 	env.mockedState.EXPECT().GetLastAccepted().Return(parentID).AnyTimes()
 	env.mockedState.EXPECT().GetTimestamp().Return(chainTime).AnyTimes()
-	onParentAccept.EXPECT().Config().Return(env.config, nil).AnyTimes()
```

Comment:
can you please add a comment regarding this removal?

Reply:
Seems that its very old leftover. It wasn't noticed cause of `.AnyTimes()` which allows this method to not be called at all.
This PR removes it, cause it also removes `MockState.Config`